app/settings.php

```diff
@@ -81,6 +81,16 @@
                         'donationTipAmount',
                     ],
                 ],
+                [
+                    'templateKey' => 'donor-funds-thanks',
```

Comment:
this template is already present and tested at https://github.com/thebiggive/mailer/blob/develop/tests/templates/DonorFundsThanksTest.php#L21